File: specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2022-08-01/examples-java/ApiManagementListNotifications.java
/** Samples for Notification ListByService. */
public final class Main {
    /*
     * x-ms-original-file: specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2022-08-01/examples/ApiManagementListNotifications.json
     */
    /**
     * Sample code: ApiManagementListNotifications.
     *
     * @param manager Entry point to ApiManagementManager.
     */
    public static void apiManagementListNotifications(
        com.azure.resourcemanager.apimanagement.ApiManagementManager manager) {
        manager.notifications().listByService("rg1", "apimService1", null, null, com.azure.core.util.Context.NONE);
    }
}

File: specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2022-08-01/examples-js/ApiManagementListNotifications.js
const { ApiManagementClient } = require("@azure/arm-apimanagement");
const { DefaultAzureCredential } = require("@azure/identity");

/**
 * This sample demonstrates how to Lists a collection of properties defined within a service instance.
 *
 * @summary Lists a collection of properties defined within a service instance.
 * x-ms-original-file: specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2022-08-01/examples/ApiManagementListNotifications.json
 */
async function apiManagementListNotifications() {
  const subscriptionId = process.env["APIMANAGEMENT_SUBSCRIPTION_ID"] || "subid";
  const resourceGroupName = process.env["APIMANAGEMENT_RESOURCE_GROUP"] || "rg1";
  const serviceName = "apimService1";
  const credential = new DefaultAzureCredential();
  const client = new ApiManagementClient(credential, subscriptionId);
  const resArray = new Array();
  for await (let item of client.notification.listByService(resourceGroupName, serviceName)) {
    resArray.push(item);
  }
  console.log(resArray);
}
